README.md

```diff
@@ -40,7 +43,7 @@ A custom CLI built for the purpose of installing and managing SolidJS apps and p
   - [x] Enable Adapters
   - [x] Enable SSR/CSR/SSG mode
 
-# CLI Design
+## Structure
```

Comment:
I couldn't decide between "Design" and "Structure" but I think we should drop the "CLI" part for sure.